src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossGridSources.scala

```diff
@@ -24,8 +24,8 @@ case class TreeLossGridSources(gridTile: GridTile, kwargs: Map[String, Any]) ext
 
   val grossCumulAbovegroundRemovalsCo2: GrossCumulAbovegroundRemovalsCo2 = GrossCumulAbovegroundRemovalsCo2(gridTile, kwargs = kwargs)
   val grossCumulBelowgroundRemovalsCo2: GrossCumulBelowgroundRemovalsCo2 = GrossCumulBelowgroundRemovalsCo2(gridTile, kwargs = kwargs)
-  val grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2e = GrossEmissionsNonCo2Co2e(gridTile, kwargs = kwargs)
-  val grossEmissionsCo2eCo2Only: GrossEmissionsCo2OnlyCo2e = GrossEmissionsCo2OnlyCo2e(gridTile, kwargs = kwargs)
+  val grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2eBiomassSoil = GrossEmissionsNonCo2Co2eBiomassSoil(gridTile, kwargs = kwargs)
```

Comment:
Why is this biomass soil now?

Reply:
Those emissions are called biomasssoil now because I added three new layers that are emissions for soil_only (as opposed to biomass + soil emissions). So I added biomasssoil to distinguish them from the soil_only emissions. In the carbonflux package, I need to distinguish between biomasssoil and soil_only but treecoverloss only uses biomasssoil. Is it okay to leave this change for clarity, though?